src/pal/src/cruntime/misc.cpp

```diff
@@ -293,23 +293,22 @@ void PAL__mm_setcsr(unsigned int i)
 
 #endif // _AMD64_ 
 
-#if defined(_DEBUG)
```

Comment:
And one can actually use it in a release build because the declaration is debug only.